components/logo-carousel/logo-carousel.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {TweenLite, TimelineLite, TimelineMax}  from 'gsap';
+
+class LogoCarousel extends React.Component {
+	constructor() {
```

Comment:
instead of setting animation speed on an attribute in HTML you can do this:
`constructor(props) {
this. animationSpeed = props. animationSpeed;
}`